README.md

```diff
@@ -118,47 +121,105 @@ your editor right next to your code.
 
 ### Updating the docs
 
-The `react-magma-docs` is the project for the documentation site. Any changes to the public API of an existing component or the creation of a new component should be documented here.
+The `react-magma-docs` is the project for the documentation site. Any changes to the public API of an existing component or the creation of a new component **must** be accurately captured documented here.
 
 This project uses [MDX](https://mdxjs.com/), allowing the combination of Markdown and React components.
 
-Each component has it's own `.mdx` file. If you have created a new component you will need to create a new `.mdx` file under the `pages/api` folder. To import your new component go to the `layout.js` file, import your new component, then add it to the `scope` prop array of the `LiveProvider`. Now you are able to use the component in your newly created `.mdx` file.
+Each component has its own `.mdx` file. If you have created a new component you will need to create a new `.mdx` file under the `pages/api` folder. If you have updated an existing component, you will need to review the existing docs page
+and update it as needed.
+
+#### Creating a docs page
 
-Be sure to add in the navigation details at the top of the page in the front matter.
+Add in the navigation details at the top of the page in the front matter.
 
 ```yaml
 ---
+pageTitle: '{ComponentName} API'
 title: '{ComponentName}'
-order: 2
+props:
+  - {ComponentName}Props
 ---
-
 ```
 
-and import a link to the design guidelines with the correct route
+Import the `DocsHeading` and `ScopeableLayout` components. You may also need other things, such as components from `react-magma-dom` to use outside of examples (such as an `Alert`), or the`Link` from `gatsby` to link to other pages.
 
 ```js
 import { DocsHeading } from '../../components/DocsHeading';
+import { ScopeableLayout } from '../../components/layout';
+```
 
-<DocsHeading to="/design/{component}/" type="design">
+Use the `DocsHeading` component to create a page heading and link to the respective Design Guidelines page if one exists.
```

Comment:
we can now do this with markdown: 

`[{ComponentName}]("/design/{component} 'api')`